File: frontend/src/types/clinic.ts
export interface Clinic {
  id: string;
  name: string;
  description: string;
  imageUrl: string;
  slug: string;
  city: string;
  address: string;
  createdAt: string;
  updatedAt: string;
}

export interface CreateClinic {
  name: string;
  description: string;
  imageUrl: string;
  city: string;
  address: string;
}

export interface UpdateClinic extends CreateClinic { }

File: backend/Models/Clinic.cs
namespace backend.Models
{
    public class Clinic
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = null!;
        public string Slug { get; set; } = null!;
        public string Description { get; set; } = null!;
        public string? ImageUrl { get; set; }
        public string City { get; set; } = null!;
        public string Address { get; set; } = null!;
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
